content/documentation/staging/tutorial/03-first-steps/03-open-application-to-outside.adoc

```diff
@@ -0,0 +1,97 @@
+---
+title: Open Travels Demo to outside traffic
+weight: 33
+---
+:icons: font
+:sectlinks:
+
+The *control* workload has now an Istio sidecar proxy injected but this application is not accessible from the outside.
+
+In this step we are going to expose the *control* service using an Istio Ingress Gateway which will map a path to a route at the edge of the mesh.
+
+NOTE: *[1]* Create a DNS entry for the *control* service associated with the External IP of the Istio Ingress
+
+icon:bullhorn[size=1x]{nbsp} This step depends on the platform you have choose, please review link:https://istio.io/latest/docs/setup/getting-started/#determining-the-ingress-ip-and-ports[Determining the Ingress IP and Ports] for more details.
+
+icon:bullhorn[size=1x]{nbsp} There are multiple ways to create a DNS entry depending of the platform, servers or services that you are using.
+
+For minikube we will check the External IP of the Ingress gateway:
+
+[source,bash]
+----
+$ kubectl get services/istio-ingressgateway -n istio-system
+NAME                   TYPE           CLUSTER-IP     EXTERNAL-IP    PORT(S)                                                                      AGE
+istio-ingressgateway   LoadBalancer   10.101.6.144   10.101.6.144   15021:30757/TCP,80:32647/TCP,443:30900/TCP,31400:30427/TCP,15443:31072/TCP   19h
+----
+
+And we will create a simple entry in the `/etc/hosts` of the tutorial machine with the desired DNS entry:
+
+[source,bash]
+----
+...
+10.101.6.144 control.travel-control.istio-cluster.org
+...
+----
+
+Then from this machine, the url `control.travel-control.istio-cluster.org` will resolve to the External IP of the Ingress Gateway of Istio.
+
+NOTE: *[2]* Use the Request Routing Wizard on *control* service to generate a traffic rule
+++++
+<a class="image-popup-fit-height" href="/images/tutorial/03-03-service-actions.png" title="Request Routing Wizard">
+    <img src="/images/tutorial/03-03-service-actions.png" style="display:block;margin: 0 auto;" />
+</a>
+++++
+
+{nbsp} +
```

Comment:
Something is missing here I guess. Perhaps: `Click on "Add Rule" so that any request will be routed to the *control* workload.`